internal/ent/schema/organization.go

```diff
@@ -32,15 +32,38 @@ func (Organization) Fields() []ent.Field {
 			NotEmpty().
 			Annotations(
 				entgql.OrderField("name"),
+				entgql.Skip(entgql.SkipWhereInput),
+			),
+		field.String("description").
+			Comment("An optional description of the Organization").
+			Optional().
+			Annotations(
+				entgql.Skip(entgql.SkipWhereInput),
+			),
+		field.UUID("parent_organization_id", uuid.UUID{}).
+			Comment("The ID of the parent organization for the organization.").
+			Annotations(
+				entgql.Type("ID"),
+				entgql.Skip(entgql.SkipWhereInput, entgql.SkipMutationUpdateInput, entgql.SkipType),
```

Comment:
Any reason why we wouldn't want to query organization based on `parent_organization_id`? 
```suggestion
				entgql.Skip(entgql.SkipMutationUpdateInput, entgql.SkipType),
```